common/src/main/java/xyz/wagyourtail/jsmacros/client/api/library/impl/FWorld.java

```diff
@@ -88,6 +93,42 @@ public boolean isWorldLoaded() {
         return mc.world != null;
     }
 
+    /**
+     * Example usage: <br>
+     * {@code World.getWorldScanner(World.getBlockAnyFilter("CONTAINS").addOption("diamond_ore", "redstone_ore"), null)}.
+     *
+     * @return a scanner for the current world
+     *
+     * @since 1.6.4
+     */
+    public WorldScanner getWorldScanner(Function<Block, Boolean> blockFilter, Function<BlockState, Boolean> stateFilter) {
+        return new WorldScanner(mc.world, blockFilter, stateFilter);
+    }
+
+    /**
+     * Returns {@code true} if one of the tests is successful.
+     *
+     * @param filterName
+     * @return a String filter to use with {@link WorldScanner}
+     *
+     * @since 1.6.4
+     */
+    public AnyStringFilter<?> getBlockAnyFilter(String filterName) {
```

Comment:
I'd suggest using world scanner as a "builder" so you'd instantiate world scanner without any filters then do
`scanner.withBlockAnyFilter(...)...scan(...)`

Reply:
That's a great idea. I will add it later.